nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfigEdit.js

```diff
@@ -855,15 +870,28 @@ export function RanEdit(props: Props) {
                 />
               }>
               {enbsCtx?.state &&
-                Object.keys(enbsCtx.state.enbInfo).map(enbSerial => (
-                  <MenuItem key={enbSerial} value={enbSerial}>
-                    <Checkbox checked={connectedEnodebs.includes(enbSerial)} />
-                    <ListItemText
-                      primary={enbsCtx.state.enbInfo[enbSerial].enb.name}
-                      secondary={enbSerial}
-                    />
-                  </MenuItem>
-                ))}
+                Object.keys(enbsCtx.state.enbInfo).map(enbSerial => {
+                  if (
+                    isEnodebUnregistered(
```

Comment:
I think we can eliminate this function and simply have a straightforward check here
`props.gateway.connected_enodeb_serials?.includes(enb.serial)`. Am I missing anything ?

Reply:
Your logic is used when checking if the current gateway includes the enodeb and put a checkmark on it if so. However, the function(the one I wrote) is used to make sure that when displaying list of enodebs to be registered to a gateway, we only list enodebs that are not registered by another gateway. In short this function makes sure that an enodeb can not be registered by more than one access gateway. So, if the other gateways do not include the enodeb serial, then that means it is free to be registered since it is not registered by other access gateway. But if we are just checking if the current gateway includes the enodeb serial and list those, then that means we are only listing enodebs that are already registered under the current gateway(and not listing the free/unregistered ones)

Reply:
Sounds good. Thanks for the clarification